### DISCUSSION TITLE
mkdocs-material/blog/2022/05/05/chinese-search-support/

### DISCUSSION THREAD

**Discussion by giscus** [squidfunk/mkdocs-material#4471] [_]
# mkdocs-material/blog/2022/05/05/chinese-search-support/

Insiders adds Chinese language support for the built-in search plugin – a  feature that has been requested many times

https://squidfunk.github.io/mkdocs-material/blog/2022/05/05/chinese-search-support/

**Comment by d3vw** [comment]
Thank u

**Comment by bigbugboy** [comment]
谢谢！

**Comment by DeepSeaCat** [comment]
nice

**Comment by Gavin0x0** [comment]
感谢

**Comment by badbye** [comment]
“搜索” matches no documents, your blog title contains this word.

**Reply by squidfunk** [reply]
That's correct. As segmentation is done by `jieba`, it's a good idea to report this to the maintainers of the project. Note that you can tune segmentation easily by providing custom dicts, see the discussion in #3915.

**Comment by YiSunYiMao** [comment]
thank you

**Comment by makerjackie** [comment]
lunr-language now support `zh`! so we just need to set `zh` in mkdocs.yaml config!  @squidfunk
https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/#multiple-languages

**Reply by squidfunk** [reply]
Please see https://github.com/mkdocs/mkdocs/issues/2509#issuecomment-882689383

**Reply by makerjackie** [reply]
I see, my bad

**Comment by Dup4** [comment]
谢谢

**Comment by w1ndys** [comment]
谢谢你的中文支持！
Mkdocs-，material真的很好用😭

**Comment by davidzhquan** [comment]
很好！！！
